rel-eng/tito.props

```diff
@@ -9,3 +9,12 @@ blacklist =
 [katello-nightly-rhel7]
 disttag = .el7
 blacklist =
+
+[katello-nightly-fedora20]
+disttag = .fc20
+whitelist = katello-repos
+
+[katello-nightly-fedora21]
+disttag = .fc21
+whitelist = katello-repos
```

Comment:
Wouldn't we want to blacklist `katello-repos` so that only `katello-client-repos` builds?